server/config/auth.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Don't add secret information in git history.
Use `.env` for local machine or Config Variables on heroku.

And also, Maybe you should recreate facebook application because these information have already recorded in git history, which means everyone can read **secret information.**

Reply:
@duyoji 
This is noted. I will use `.env`
Thanks for the notification

Reply:
When you use local machine to check your login feature, you can use `.env`.

but, when we use it on the Internet, you need to set Config Variables. because `.env` is ignored by `.gitignore`.

Reply:
@duyoji Done setting for both heroku local